Checksum Mismatch MacOS 1.18.3

When trying to install on MacOS AMD64 Getting the following error:

```
BUILD FAILED (OS X 11.6.6 using go-build 2.0.0beta1)

checksum mismatch: Go Darwin 64bit 1.18.3.tar.gz (file is corrupt)

expexted a23a24c5528671d444328a36a98056902f699a5a211b6ad5db29ca0c012e0085, got d9dcf8fc35da54c6f259be41954783a9f4984945a855d03a003a7fd6ea4c5ca1
```

--- a/plugins/go-build/share/go-build/1.18.3
+++ b/plugins/go-build/share/go-build/1.18.3
@@ -1,4 +1,4 @@
-install_darwin_64bit "Go Darwin 64bit 1.18.3" "https://golang.org/dl/go1.18.3.darwin-amd64.tar.gz#a23a24c5528671d444328a36a98056902f699a5a211b6ad5db29ca0c012e0085"
+install_darwin_64bit "Go Darwin 64bit 1.18.3" "https://golang.org/dl/go1.18.3.darwin-amd64.tar.gz#d9dcf8fc35da54c6f259be41954783a9f4984945a855d03a003a7fd6ea4c5ca1"
 
 install_darwin_arm "Go Darwin arm 1.18.3" "https://golang.org/dl/go1.18.3.darwin-arm64.tar.gz#40ecd383c941cc9f0682e6a6f2a333539d58c7dea15c842434d03afafe2f7242"
 

